utils/cp_utils.py

```diff
@@ -60,14 +60,19 @@ def run_cellprofiler(
         multiple sets of images (e.g., per plate) so that the outputs will have different names (default is None)
         analysis_run (bool, optional): will use functions to complete an analysis pipeline (default is False)
     """
-    # check to make sure all paths are correct and the directory exists before 
+    # check to make sure paths to pipeline and images are correct before running the pipeline
     if not os.path.exists(path_to_pipeline):
```

Comment:
```suggestion
    if not pathlib.Path(path_to_pipeline).exists():
```
Replacing `os` with `pathlib.Path`. Feel free to remove this suggestion. 

Also, it seems you're only checking if the directory exists. However, I do not see if you're checking if the provided paths is a directory. 

If you plan on doing that, you can also write `pathlib.Path.is_dir()`, Here's the [documentation](https://docs.python.org/3/library/pathlib.html#pathlib.Path.is_dir) 

Feel free to reject this as well if you think if a non-directory path is not going to be passed through

Reply:
I was more going for that it needs to be a directory with the images and it needs to be a correct path to a file for the pipeline. I will make the changes to make this more obvious. See [e78c1b7](https://github.com/WayScience/nf1_cellpainting_data/pull/7/commits/e78c1b74d4484beceb20ad08b509a52ab629832a) for change.